Create 0000-01-02-shubila.md (#3)
shubila/github-slideshow: _posts/0000-01-02-shubila.md

---
layout: slide
title: "Welcome to our second slide!"
---

> I hope this is right because now I am not sure if I know what I am doing.

Author: ```suggestion
I feel now even more confused as to what I am doing........
```
```suggestion
I hope this is right because now I am not sure if I know what I am doing.
```
